etrago/appl.py

```diff
@@ -405,7 +408,7 @@ def run_etrago(args, json_path):
         etrago.network.generators.carrier=='CH4']+= 25.6+0.201*76.5
 
     # ehv network clustering
-    etrago.ehv_clustering()
+    # etrago.ehv_clustering()
```

Comment:
Why did you comment the ehv_clustering?

Reply:
My bad, I did it for testing, because ehv_clustering took hours (it seemed to be stuck after asking for number of CPUs). I uncommented it again [cb5c70d](https://github.com/openego/eTraGo/pull/403/commits/cb5c70d702c6d00176d235ac2269aab7a358c2ad)